Makefile

```diff
@@ -144,6 +144,16 @@ create-bastion:
 delete-bastion:
 	./scripts/delete-stack.sh network/bastion "$(config_path)" "$(stack_prefix)"
 
+################################################################################
```

Comment:
Don't forget to update changelog with this addition change.

Reply:
Change log updated as requested.